### ISSUE TITLE
Independent Auth Guards: Add note about column types when using MySQL database & `database` session driver

### ISSUE THREAD

**Issue by duncanmcclean** [statamic/docs#1475]
When you're using [independent auth guards](https://statamic.dev/tips/using-an-independent-authentication-guard) with Statamic, along with a MySQL database and the `database` session driver (which is the new default in Laravel 11), you may need to change the type of the `user_id` column to a varchar to account for Statamic user IDs.

**Comment by danielreales7** [comment]
Could this be related to the fact that when trying to obtain the logged in user the user variable is always empty?

I've been working on this all day and it never returns the user value, however it detects the variable {{ user }} because it is inside but it doesn't return the name:

```php
{{ if logged_in }}
  {{ user }}
  Welcome back! {{ email }}.
  {{ /user }}
{{ /if }}
```

I'm using: [independent auth guards](https://statamic.dev/tips/using-an-independent-authentication-guard) with Laravel 11.

Authentication works perfectly for web guards and statamic for the cp.

**Comment by duncanmcclean** [comment]
Possibly? 

Are you using the `database` session driver? If you are, changing the column type should fix the issue. Otherwise, it'll be a separate issue.

**Comment by danielreales7** [comment]
No, it doesn't work. I create a new issue [https://github.com/statamic/cms/issues/11438](url)

**Comment by igmltd** [comment]
experiencing the same issue on production. statamic keep clearing the user_id on the sessions table even though its set to varchar 255

Note it works if I change the session to file, however we need the db. thanks.

**Comment by hassan402-paymentrequired** [comment]
I'm also facing the same issue @igmltd, has any one find the solution
